### PR TITLE
Omit analysis.

### PR DESCRIPTION
**Pull request by vext01**
OK?

### CODE REVIEW

**Comment by ltratt** [comment]
Can we not just delete the commented out lines?

**Comment by vext01** [comment]
If you like. It's easier to see what we did before without digging in vcs. As you wish.

**Comment by ltratt** [comment]
Commented out lines make future code reading harder, and that's more important than anything else IMHO. Kill 'em, I say!

**Comment by vext01** [comment]
splat.

**Comment by vext01** [comment]
Please don't merge until Sarah and CF have commented.

**Comment by vext01** [comment]
Remove another function related to rolling stats.

File: mk_graphs2x2.py [hunk 1]
```diff
@@ -73,29 +63,6 @@ def main(mode, data_dcts, mch_names, x_bounds):
...
-
-    # Must come after std_dev lines built
-    avg = np.insert(avg, 0, [None] * pad)
-
-    axis.plot(lower_std_dev)
-    axis.plot(upper_std_dev)
-    axis.plot(avg)
-
-    #axis.plot(avg * (1 + LINES_PERCENT / 100.0))
-    5#axis.plot(avg * (1 - LINES_PERCENT / 100.0))
```
Comment (cfbolz) [inline mk_graphs2x2.py:98]: what is the weird 5 here?

Comment (cfbolz) [inline mk_graphs2x2.py:98]: Ah, you are deleting it anyway, so fine

Comment (vext01) [inline mk_graphs2x2.py:98]: Yeah looks like a typo.

**Pull request merged**